src/components/common/Extensions/UserHub/partials/NotificationsTab/partials/Notification/Notification.tsx

```diff
@@ -83,23 +86,34 @@ const Notification: FC<NotificationProps> = ({
   }
 
   // If the notification type is an expenditure update:
-  if (
-    [
-      NotificationType.ExpenditureReadyForReview,
-      NotificationType.ExpenditureReadyForFunding,
-      NotificationType.ExpenditureReadyForRelease,
-      NotificationType.ExpenditureFinalized,
-      NotificationType.ExpenditureCancelled,
-    ].includes(notificationType)
-  ) {
-    return (
-      <ExpenditureNotification
-        colony={notificationColony}
-        isCurrentColony={isCurrentColony}
-        loadingColony={loadingColony}
-        notification={notification}
-      />
-    );
+  // Or a motion supporting an expenditure (eg. for funding)
+  if (hasExpenditureId) {
+    if (
+      [
+        NotificationType.MotionCreated,
+        NotificationType.MotionOpposed,
+        NotificationType.MotionSupported,
+        NotificationType.MotionVoting,
+        NotificationType.MotionReveal,
+        NotificationType.MotionFinalized,
+        NotificationType.ExpenditureReadyForReview,
+        NotificationType.ExpenditureReadyForFunding,
+        NotificationType.ExpenditureReadyForRelease,
+        NotificationType.ExpenditureFinalized,
+        NotificationType.ExpenditureCancelled,
+      ].includes(notificationType)
+    ) {
+      return (
+        <ExpenditureNotification
+          colony={notificationColony}
+          isCurrentColony={isCurrentColony}
+          loadingColony={loadingColony}
+          notification={notification}
+        />
+      );
+    }
+    // if isExpenditure is true but we have a mention notification we'll not display it
+    return null;
```

Comment:
Can we just display a mention here in the (hopefully never going to happen) case that we reach this code? Since it'd be very odd for a user to see that they have an unread notification, but then not seeing anything rendered in the notification list.